swift/ql/src/diagnostics/SuccessfullyExtractedLines.ql

```diff
@@ -0,0 +1,15 @@
+/**
+ * @name Successfully extracted lines
+ * @description Count all lines in source code in which something was extracted.
```

Comment:
The QL is actually counting the number of lines of source code that are the start location of something that was extracted - so for example a multi-line comment would only be counted as one line.  I'm not sure if this is worth clarifying.

Reply:
excellent point, I was [concurrently thinking the same](https://github.com/github/codeql/pull/12725#discussion_r1154171719), editing a previous comment.

Yeah, probably it's better to clarify this in the description.

Reply:
Done :slightly_smiling_face: